### PR TITLE
feat: fixed duplicate timezones

### PR DESCRIPTION
**Pull request by theijhay**
​

## Description

This update addresses an issue where duplicate time zone entries appeared in the dropdown menu on the Language & Region settings page of the Admin Dashboard. The service logic has been refactored to ensure that only unique time zones are retrieved and displayed in the dropdown.

​

## Related Issue (Link to issue ticket)

This pull request resolves the issue related to duplicate time zone entries in 
the dropdown, as tracked in issue #943 

​

## Motivation and Context

The change was necessary to eliminate confusion among users when selecting a time zone from the dropdown. Duplicates in the list could lead to errors or user frustration. By ensuring each time zone is listed only once, the user experience is improved, and potential errors are reduced.

​

## How Has This Been Tested?

The changes were tested using Postman to verify the `api/v1/regions/` endpoint returns only unique time zones without duplicates. The testing included querying the database to ensure that the list is comprehensive and accurate

​

## Screenshots (if appropriate - Postman, etc):

Screenshot from 2024-08-24 12-37-36

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
      ​

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: api/v1/routes/regions.py [hunk 1]
```diff
@@ -60,4 +60,17 @@ def update_region(region_id: str, region: RegionUpdate, db: Session = Depends(ge
 @regions.delete("/{region_id}", status_code=status.HTTP_204_NO_CONTENT)
 def delete_region(region_id: str, db: Session = Depends(get_db)):
     region = region_service.delete(db, region_id)
-    return
\ No newline at end of file
+    return
+
+
+@regions.get(
+    '/', response_model=List[str])
```
Comment (johnson-oragui) [inline api/v1/routes/regions.py:67]: Remove the extra slash.

Comment (johnson-oragui) [inline api/v1/routes/regions.py:67]: Remove the trailing slash

File: api/v1/routes/regions.py [hunk 2]
```diff
@@ -60,4 +60,17 @@ def update_region(region_id: str, region: RegionUpdate, db: Session = Depends(ge
...
+def get_unique_timezones(db: Session = Depends(get_db)):
```
Comment (johnson-oragui) [inline api/v1/routes/regions.py:68]: No pytest for the endpoint.

**Review by johnson-oragui — CHANGES_REQUESTED** [review]
Please add a test, and remove the trailing slash

File: api/v1/routes/regions.py (hunk 2 above)
Comment (theijhay) [reply api/v1/routes/regions.py]: @johnson-oragui  changes has been made

**Review by joboy-dev — APPROVED** [review]

**Review by joboy-dev — CHANGES_REQUESTED** [review]
Where are the tests Johnson requested for?

**Comment by theijhay** [comment]
> Where are the tests Johnson requested for?

@joboy-dev This is just a fix, and perhaps there is no test for regions endpoints, the person didn't write it.
and you want me to write test for all these endpoints?

**Review by joboy-dev — APPROVED** [review]

**Review by johnson-oragui — APPROVED** [review]

**Pull request merged**